controllers/executability/executability.js

```diff
@@ -17,13 +17,24 @@ exports.getBadgeFromData = (req, res) => {
         res: res
     };
 
+    let service = config.executable.mainService;
+    //ToDo: Implement multiple services and a fallback when there is no result
+    let allServices = config.executable.services;
+    if (allServices.indexOf(service) !== -1) {
+        debug('Using service %s for badge %s', service, passon.id);
+        //ToDo: Return a different promise based on the service
+    } else {
+        debug('No service for badge %s found', passon.id);
+        res.status(404).send('{"error":"no service for this type found"}');
```

Comment:
Should the function not `return` here?

Recommend to switch the condition and subsequently skip the else clause.